addons/project/views/project_views.xml

```diff
@@ -905,7 +905,7 @@
                             <field name="display_project_id" string="Project" attrs="{'invisible': [('parent_id', '=', False)]}" domain="[('active', '=', True), ('company_id', '=', company_id)]"/>
                             <field name="user_ids"
                                 class="o_task_user_field"
-                                options="{'no_open': True}"
+                                options="{'no_open': True, 'no_quick_create': True}"
```

Comment:
As user_id is no more present, I take the opportunity to change a bit the commit message:
```txt
[IMP] project, sale_{project, timesheet}: improve user experience

This commit introduces some user experience improvements:

    - Always display sale_line_id field whether partner_id is set or not.
    - Don't restrict sale orders to the partner_id's one
    - Set partner_id to the one of the SOL if partner_id is False
    - Remove create option for fields 'user_ids'
    - Display the kanban state in the kanban view even in fsm

task-2585357

```

Reply:
Done :+1: